src/idl_gen_go.cpp

```diff
@@ -385,9 +385,7 @@ class GoGenerator : public BaseGenerator {
       code += "\t\treturn ";
     }
     code += CastToEnum(field.value.type, getter + "(o + rcv._tab.Pos)");
-    if (field.IsScalarOptional()) {
```

Comment:
I think we've been using different formatter configs somehow. 

I've been using `git-clang-format -f` (though I'm not sure what config that resolves to)

Reply:
I always have difficulties with the clang format on my machine. If I do a PR on my work computer (like this) it uses a google version of clang-format that does something different. This is on my mental todo list to fix.